packages/hash/api/src/model/util.ts

```diff
@@ -148,7 +150,7 @@ export const generateWorkspacePropertyTypeSchema = (params: {
     namespaceUri: workspaceTypesNamespaceUri,
     title: params.title,
     kind: "propertyType",
-  }),
```

Comment:
Can we actually replace this cast now? I think we should modify `generateSchemaVersionedUri` to return the correct type. I'm guessing we can't easily call `isVersionedUri` because of the node/web problem? 🙃

Reply:
Adding `as const` in the `generateSchemaVersionedUri` made the cast redundant :+1: 

c5fdb196fa10666131dd10aff5d7973af9cce7ba